### PR TITLE
Visual bug fixes

### PR DESCRIPTION
**Pull request by erikjalevik**
This PR addresses the following Trello tickets:

* In Safari, only the right carousel arrow appears above the image
* Text overruns carousel at certain widths
* Broken footer layout in certain width
* Contact component has weird layout in certain widths

### CODE REVIEW

**Comment by sarunast** [comment]
Deploy preview for *masifunde* ready!

Built with commit f23cb5d8cd2a5bfdc6ea71d6bdbbbac0143b1b69

https://deploy-preview-225--masifunde.netlify.com

File: components/Carousel.js [hunk 1]
```diff
@@ -88,28 +90,23 @@ const StyledCarousel = styled(Carousel)`
...
-
-    button {
-      opacity: 0 !important;
-    }
   }
 
-  .slider-decorator-0 {
-    transform: rotateY(180deg) translateY(-50%) !important;
+  :focus {
+    outline: 0;
```
Comment (denisw) [inline components/Carousel.js]: This seems terrible accessibility-wise. Can't you add another focus indicator?

Comment (erikjalevik) [reply components/Carousel.js]: Added a basic one, but these buttons don't even show until you hover over the component so it doesn't improve things much.

Comment (denisw) [reply components/Carousel.js]: Ah, I see... I guess that needs a more thorough solution then (if it's even possible with that carousel plugin). Perhaps we need to live with keyboard / screen reader users only having access to the first carousel page then, as much as it sucks.

**Pull request merged**